node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -3642,7 +3828,7 @@ mod tests {
 
         <------------------------------------->
          */
-
+        let before = time_t_timestamp();
```

Comment:
I do think this should tightly surround the "act" of the test. It shouldn't be placed so far away. Please put it back to its former place.